app/identifiers/register/trustees/individual/TrusteeNameId.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package identifiers.register.trustees.individual
+
+import identifiers._
+import identifiers.register.trustees.TrusteesId
+import models.person.PersonName
+import play.api.libs.json.JsPath
+
+case class TrusteeNameId(index: Int) extends TypedIdentifier[PersonName] {
```

Comment:
There seems to be a [`TrusteeDetailsId`](https://github.com/hmrc/pensions-scheme-frontend/blob/master/app/identifiers/register/trustees/individual/TrusteeDetailsId.scala) is this the same as that?

Reply:
One's the old ID, trusteenameid is the new one with only name (not date)

Reply:
Do you mean the inner type of the TypedIdentifier?

Reply:
No, there are 2 ids, one for the new name page and one for the old one which includes date of birth field as well I believe.